src/fluree/db/conn/file.cljc

```diff
@@ -76,7 +84,22 @@
              (System/exit 1))))
        (catch Exception e (throw e)))
      :cljs
-     (throw (ex-info "TODO: implement for node" {}))))
+     (try*
+       (fs/writeFileSync path val)
+       (catch* e
+               (if (= (.-code e) "ENOENT")
```

Comment:
I want to find a way to do this consistently and automatically across editors (if even possible), but it would be great to indent these just like `catch`. Not a big deal either way, though.